pkg/scalers/nats_jetstream_scaler.go

```diff
@@ -21,6 +21,8 @@ import (
 const (
 	jetStreamMetricType          = "External"
 	defaultJetStreamLagThreshold = 10
+	natsHTTPProtocol             = "http"
+	natsHTTPSProtocol            = "https"
```

Comment:
Don't use HTTP and HTTPS in upper case, 
```suggestion
	natsHttpProtocol             = "http"
	natsHttpsProtocol            = "https"
```